ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.cpp

```diff
@@ -72,7 +72,7 @@ Tensor optimized_conv_new(const Tensor& a, const Tensor &b, std::optional<const
     bool enable_subblock_padding,
     bool use_non_tile_height
 ) {
-    std::vector<Tensor> output_tensors = {Tensor(operation::get_workers_for_op_output({a, b}))};
```

Comment:
should this namespace thing be decoupled from sweep changes?

Reply:
i added that by mistake and then my internet died before i could fix it, please ignore that

Reply:
If you remove that you will need less people to re-view your change (less files changed less code owners triggered)

Reply:
I would close this one and open new PR to get less people involved.